_data/header.yaml

```diff
@@ -0,0 +1,13 @@
+type: static
+links:
+  - title: Browse AIPs
+    relativeUrl: /general
```

Comment:
We should use `snake_case` since Jekyll's built in config options (e.g. `redirect_from`, `relative_url`, etc.) also do.

Reply:
It's actually unclear to me what convention Jekyll uses. There is also `baseurl`, so...

But it looks like you make this change under the basis of Ruby's standard, which I think makes more sense.